### DISCUSSION TITLE
Why is it challenging to add off-ramp to Rainbow wallet?

### DISCUSSION THREAD

**Discussion by RonanKMcGovern** [rainbow-me/rainbowkit#813] [Q&A] [answered]
The on-ramp experience on Rainbow is phenomenal using Apple Pay.

I'm curious what the challenges are with off-ramp?

I would have thought off-ramp is less risky because crypto is being sent by the wallet (which is irreversible) and fiat is being received. Surely the on-ramp is riskier because the crypto can't easily be claimed back by the on-ramp provider?

Thanks, Ronan

**Comment by nickbytes** [comment, marked as answer]
Hi @RonanKMcGovern – I'm going to close this issue discussion for being somewhat off-topic with regards to RainbowKit. There are many challenges in on-ramps and off-ramps including (but not limited to): KYC/AML, liquidity and reserve management, payment processing, delivery of funds, unclear & changing regulation, fees, etc etc. 

We're currently working to improve our on-ramp in the wallet, stay tuned! Appreciate you Rainbow and RainbowKit!
